odoo/addons/base/models/assetsbundle.py

```diff
@@ -722,6 +817,21 @@ def to_node(self):
                 ['data-asset-version', self.bundle.version],
             ]), self.with_header())
 
+    def with_header(self, content=None, minimal=True):
+        if minimal:
+            return super(JavascriptAsset, self).with_header(content)
+        else:
+            header = [
+                "*  Filepath: %s" % self.url,
+                "*  Bundle: %s" % self.bundle.name,
+                "*  Lines: %s" % len(content.splitlines()),
+            ]
+            length = max([len(line) for line in header])
+            header = [line + " " * (length - len(line) + 2) + "*" for line in header]
```

Comment:
Python's string formatting can do this crap for you, without the risk of underflow:
```suggestion
            header = [f'{line:<{length + 2}}*' for line in header]
```
I'm not sure my increment is correct as I don't really know where the `+2` comes from.

Reply:
Incidentally since you compute `length + 2` multiple times and never use `length` naked, why not just add the 2 when defining length?